include/panel_state_manager.hpp

```diff
@@ -84,10 +80,13 @@ class PanelStateManager
 
     /**
      * @brief Constructor.
-     * @param[in] transport - transport object to call transport functions.
+     * @param[in] transport - transport object to call transport functions
+     * @param[in] execute - pointer to executor class.
      */
-    PanelStateManager(std::shared_ptr<Transport> transport) :
-        transport(transport)
+    PanelStateManager(std::shared_ptr<Transport> transport,
```

Comment:
why this change is not in your Gerrit ?

Reply:
I think the changes are merged in Gerrit but not here, so the difference. I'll try to change that.